docs/runbooks/secrets-reference.md

```
@@ -109,17 +109,32 @@ infra/config/secrets/
-# Or use the full generate command
-toolkit credentials generate --env <env>
+toolkit credentials hash-password apps.services.security.authelia.users_operator_password_hash --env <env>
+git add infra/config/secrets/<env>.enc.yaml && git commit    # SOPS is modified, not committed
+make apply-secrets ENV=<env>
 ```
-**What breaks**: User must know the new password to login.
+
+The command **writes the hash into SOPS itself** and never prints it (#934, fixed
+2026-09-01). Before that it printed the value and asked for a manual edit, which is
+how an argon2 hash reached a session transcript on 2026-08-31 — the exact outcome
+#934 predicted when it was filed three weeks earlier. If you are reading an older
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)